custom_components/oncharger/sensor.py

```diff
@@ -82,22 +95,53 @@ class OnchargerSensorEntityDescription(SensorEntityDescription):
     suggested_display_precision=0,
 )
 
+POWER1_DESCRIPTION = OnchargerSensorEntityDescription(
```

Comment:
Let's DRY up this and stuff below. Some tips on how to do it:

https://github.com/krasnoukhov/homeassistant-smart-maic/blob/b965e7934f88665fc5b0a9b5f587a126c3cc6038/custom_components/smart_maic/sensor.py#L35-L45

https://github.com/krasnoukhov/homeassistant-smart-maic/blob/b965e7934f88665fc5b0a9b5f587a126c3cc6038/custom_components/smart_maic/sensor.py#L126-L131